### PR TITLE
.vscode: Add tasks for each wiki and create a settings file

### PR DESCRIPTION
**Pull request by TunaLobster**
Recreation of #4556 after a IDE git mess.

### CODE REVIEW

**Comment by Hwurzburg** [comment]
@TunaLobster someone needs to review this... any suggestions?

**Comment by TunaLobster** [comment]
@bugobliterator or @hendjoshsr71 or @IamPete1 Whoever is first I guess.

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        "blimp/**/common-*": true,
+        "copter/**/common-*": true,
+        "dev/**/common-*": true,
+        "mavproxy/**/common-*": true,
+        "plane/**/common-*": true,
+        "planner/**/common-*": true,
+        "planner2/**/common-*": true,
+        "rover/**/common-*": true,
+    }
+}
```
Comment (IamPete1) [inline .vscode/settings.json:22]:
```suggestion
}

```

File: .vscode/tasks.json [hunk 2]
```diff
@@ -5,11 +5,149 @@
...
+                "rover"
+            ],
+            "group": {
+                "kind": "build",
+                "isDefault": true
+            }
+        },
     ]
-}
+}
```
Comment (IamPete1) [inline .vscode/tasks.json:153]:
```suggestion
}

```

**Review by IamPete1 — APPROVED** [review]
Basically looks good, although I have not tested.

File: .vscode/settings.json (hunk 1 above)
Comment (TunaLobster) [reply .vscode/settings.json:22]: Whoops! For some reason VSCode settings json formatter doesn't add those in automatically. I'll fix it in a few hours when I get to a desktop.

File: .vscode/settings.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{
+    "esbonio.sphinx.confDir": "${workspaceFolder}/copter/source",
+    "grammarly.config.documentDialect": "american",
+    "editor.wordWrap": "on",
```
Comment (bugobliterator) [inline .vscode/settings.json:4]: this seems like out of scope of this PR change, some devs might not want this.

**Review by bugobliterator — CHANGES_REQUESTED** [review]

File: .vscode/settings.json (hunk 3 above)
Comment (TunaLobster) [reply .vscode/settings.json]: Good catch! I tend to keep this on for the wiki, but not everyone likes word wrapping.

**Comment by TunaLobster** [comment]
Suggestions applied. Good to go?
